api/migrations/0003_fake_data.py

```diff
@@ -60,6 +60,7 @@ class RecipeStorage:
 
     def __init__(self):
         self.faker: Faker = Faker()
+        Faker.seed(0)
```

Comment:
Does this make sure the random names are always the same?

Reply:
To be honest, I'm not sure how it works :D But hope it will help to prevent problems of constraints violating.

As I understand - during every "clear" recreation it will produce same values, but of course in one "session" all values would be random.